src/backersManager/newAllocation.ts

```diff
@@ -49,5 +61,37 @@ export function handleNewAllocation(event: NewAllocationEvent): void {
   gauge.allocation_ = event.params.allocation_;
   gauge.lastBlockNumber_ = event.block.number;
   gauge.lastBlockTimestamp_ = event.block.timestamp;
+
   gauge.save();
 }
+
+function _handleBacker(event: NewAllocationEvent): void {
+  let backer = Backer.load(event.params.backer_);
+  if (backer == null) {
+    backer = new Backer(event.params.backer_);
+    backer.isBlacklisted_ = false;
+  }
+  backer.totalAllocation_ = event.params.allocation_;
+
+  backer.save();
+}
+
+function _handleBackerToBuilder(event: NewAllocationEvent): void {
+  const gaugeToBuilder = GaugeToBuilder.load(event.address);
+  if (gaugeToBuilder == null) return;
+
+  let backerToBuilder = BackerToBuilder.load(
+    event.params.backer_.concat(gaugeToBuilder.builder_)
+  );
+  if (backerToBuilder == null) {
+    backerToBuilder = new BackerToBuilder(
+      event.params.backer_.concat(event.params.gauge_)
+    );
+    backerToBuilder.backer_ = event.params.backer_;
+    backerToBuilder.builder_ = event.params.gauge_;
+    backerToBuilder.totalAllocation_ = BigInt.zero();
```

Comment:
I don't think we need to set it to zero as we're setting it just after this line

Reply:
update